plugins/lookup/index_of.py

```diff
@@ -0,0 +1,372 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+"""
+The index_of lookup plugin
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+    lookup: index_of
+    author: Bradley Thornton (@cidrblock)
+    version_added: "1.4"
+    short_description: Find the indicies of items in a list matching some criteria
+    description:
+        - This lookup returns a list of indicies of items matching some criteria in a list.
```

Comment:
If I understand correctly the return is of type list if `wantlist` is set to `True`. Should the description change accordingly.